### PR TITLE
Reorder Travis build command orders

### PR DESCRIPTION
**Pull request by penev92**
Two more tweaks in the spirit of #7294.

### CODE REVIEW

**Comment by Phrohdoh** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -6,10 +6,10 @@ language: csharp
 # Run the build script
 # call RALint to check for YAML errors
 script:
+ - make check
```
Comment (obrakmann) [inline .travis.yml]: make check invokes the Utility, which hasn't been built yet. Those two targets (make and check) should probably depend on the `utility` target.

File: Makefile [hunk 2]
```diff
@@ -172,40 +172,57 @@ OpenRA.Editor.Form1.resources:
 editor: OpenRA.Editor.MapSelect.resources OpenRA.Editor.Form1.resources $(editor_TARGET)
 
 check:
+	@echo ""
```
Comment (obrakmann) [inline Makefile]: just a plain `echo` will produce a newline, no need for the empty string.

File: Makefile [hunk 3]
```diff
@@ -172,40 +172,57 @@ OpenRA.Editor.Form1.resources:
 editor: OpenRA.Editor.MapSelect.resources OpenRA.Editor.Form1.resources $(editor_TARGET)
 
 check:
```
Comment (obrakmann) [inline Makefile]: Make this (and the `test` target below) depend on the `utility` target (see how the `docs` target does it if you don't know how that works).

**Comment by obrakmann** [comment]
Thanks. Works for me, but I'd like to wait for one of the others to look at it. I'm not sure if they had a reason to do it like it was. Anyway, :+1:

**Pull request merged**